benchmark/benchmark.toml

```diff
@@ -0,0 +1,21 @@
+# TOML document for benchmark tests
+
+title = "TOML Example"
+atom = 6.626e-34
+cash = true
+chars = ["a", "b", "?", "!", "`", "-", "  d", "\t", "\\", "\"" ]
+digits = [1.0, 2.0, nan, -nan, +inf, -inf, 1.0]
+bool = [false, true, false]
+# date = [1979-05-27]
+# today = 1979-05-27T07:32:00Z
+
+[fruit]
+name = "apple"
```

Comment:
Let's add one more field here with key `description` which will be multiline string (3 lines is enough) for apple description